### DISCUSSION TITLE
Containers getting killed by HAOS?

### DISCUSSION THREAD

**Discussion by oraculix** [alexbelgium/hassio-addons#815] [Q&A]
Hello all,
I'm using Portainer from the hassio-addons repo for about a year now (thanks for providing this, btw!). My main use case is to run CUPS on my RasPi to make my old printer available on my network.

After a recent update to HAOS 10 I saw that my "own" containers (those not belonging to HA) get terminated just seconds after they started. I tried pulling the latest images from DockerHub and creating containers from different images, alway with the same negative results.

Some searches on the net, HA forums and log files didn't yield anything usable for me. Before I start digging deeper into this: Are you aware of any recent HA Core / HAOS functionality to kill "foreign" containers?

kr, Uwe

**Comment by alexbelgium** [comment]
No info on that! You should ask on the discord, their devs are there who could know

**Reply by oraculix** [reply]
It's been a while, but I still want to leave an update here:
No solution yet, but it seems to be a common issue starting with HAOS 10. See [this thread](https://github.com/MaxWinterstein/homeassistant-addons/issues/152) on MaxWinterstein's repo, where a suggested workaround is to downgrade to HAOS 9.5. From a security perspective I don't think this is a good idea.

**Comment by alexbelgium** [comment]
You could try to start the containers using the ssh terminal add-on and the 'docker` command. It worked on my system. Perhaps that might check if the issue is with portainer or ha itself (probably the later indeed)
